guides/user/pipeline/managing-pipelines/index.md

```diff
@@ -0,0 +1,137 @@
+---
+layout: single
+title:  "Managing Pipelines"
+sidebar:
+  nav: guides
+---
+
+{% include toc %}
+
+Pipelines are the essential tool in Spinnaker for configuring how to deploy your
+application. They are composed of a series of stages that can be combined in
+almost any order, which makes them flexible, consistent, and repeatable.
+
+You can configure your pipelines to run entirely automatically or require manual
+intervention to ensure that everything is working as expected. Similarly,
+pipelines can either be automatically triggered by a wide range of external
+inputs, including other pipelines, or be manually triggered.
+
+This guide explains how to configure and control pipelines, including creating,
+adding triggers to, or disabling pipelines.
+
+## Create a pipeline
+1. Navigate to the **Pipelines** tab in your Spinnaker application.
+  ![](images/pipelines-tab.png)
+2. Click **Create**, located in the upper right corner of the Pipelines tab.
+  ![](images/create.png)
+3. Choose **Pipeline** from the drop down menu and name your pipeline.
+
+After you create your pipeline, add stages to specify the actions that your
+pipeline will perform.
+
+### Add a stage
+
+Your first stage in any pipeline will be a **Configuration** stage, where you
```

Comment:
I don't think this is technically a "stage", even though it sort-of looks like one in the UI.